docs/base/readme.md

```diff
@@ -116,58 +129,64 @@ We highly encourage you setting up Bootstrap bundle on your SourceJS instance, a
 
 Read our how-to articles, to get more information about the [SourceJS catalog set-up](https://github.com/sourcejs/blog-howto/tree/master/catalog-setup).
 
-### Specs Showcase
-
-Highlighting the variety of different ways for organizing Spec pages we gathered another special bundle. View it's source code at [showcase repo](https://github.com/sourcejs/example-specs-showcase) and compare with [rendered result](http://sourcejs.com/specs/example-specs-showcase/).
-
-Showcase includes both native Specs examples, and ones that are rendered with plugins like [sourcejs-contrib-dss](http://github.com/sourcejs/sourcejs-contrib-dss) and [sourcejs-jade](http://github.com/sourcejs/sourcejs-jade).
-
-<a href="/docs/spec" class="source_a_hl">Also check the SourceJS Spec page documentation.</a>
-
 ### Style Guide Driven Development
 
 To get more insights about recommended workflow within Style Guide platform check [this example bundle](https://www.youtube.com/watch?v=KeR8Qhgyb6M) together with short screencast.
 
-<div style="max-width: 600px;"><div style="position: relative; padding-bottom: 56.25%;">
+<div style="max-width: 100%;"><div style="position: relative; padding-bottom: 56.25%;">
 <iframe width="100%" height="100%" style="position: absolute;" src="https://www.youtube.com/embed/KeR8Qhgyb6M" frameborder="0" allowfullscreen></iframe>
 </div></div>
 
 
 ## Configuration
 
-SourceJS engine is highly configurable and allows to override almost any options from your instance set-up using personal configuration file `sourcejs/user/options.js`. All default, and safe to change options are described in base configuration file `sourcejs/options.js`.
+SourceJS engine is highly configurable. Engine and Middleware are configured in 1 place:: `options.js`.
 
 With version 0.5.3 we also introduced context level options `sourcejs-options.js`, which allows to configure any catalog specifically for your needs. Read more about it, and other configuration capabilities in [Engine Configuration](/docs/configuration) doc.
 
-## Plugins
+## Middleware
 
 As a Style Guide Platform we focus on flexibility and ease of integration. All SourceJS core modules are easy to configure and replace with your customized version.
 
 Plugins are working in the same way as core modules, but are kept outside the main platform, allowing to separate specific features.
 
 Here is a list of available plugins:
 
-* [sourcejs-slm](https://github.com/venticco/sourcejs-slm) (new)
-* [sourcejs-md-react](https://github.com/mik01aj/sourcejs-md-react) (new)
-* [sourcejs-contrib-browser-sync](https://github.com/sourcejs/sourcejs-contrib-browser-sync) (new)
-* [sourcejs-react](https://github.com/szarouski/sourcejs-react) (new)
-* [sourcejs-contrib-dss](http://github.com/sourcejs/sourcejs-contrib-dss) (new)
-* [sourcejs-spec-status](https://github.com/sourcejs/sourcejs-spec-status)
-* [sourcejs-crowd-voice](https://github.com/sourcejs/sourcejs-crowd-voice)
+### New spec languages:
```

Comment:
No need to use work "new", let just one section "Spec Language Integrations".  Newest could be just placed on top, and marked as bold.

Also, linking to npm search would be useful https://www.npmjs.com/search?q=sourcejs-